install.ps1

```diff
@@ -53,7 +57,7 @@ function download {
     $binary_error="Failed to download $file"
     return
   }
-  check_binary >$null
+  echo y | cacls $fzf_base\bin\fzf.exe /p administrator:f ; check_binary >$null
```

Comment:
* I'm seeing `NOTE: Cacls is now deprecated, please use Icacls.` message when I run cacls. Should we use icacls instead?
* Does this work even when the user account is not "administrator"?